docs/config/README.md

```diff
@@ -467,6 +467,30 @@ The module will be shown if any of the following conditions are met:
 symbol = "⚙️ "
 ```
 
+## Nim
```

Comment:
I think Nim should be in ascending order. All modules are listed in are in ascending order currently